tests/tips_views_tests.py

```diff
@@ -72,27 +83,53 @@ def test_bad_form(self, generate_bad_form):
 
 @pytest.mark.django_db
 class TestTipViews:
-    def test_tips_board_function_200(self, client):
-        response = client.get('/tips/')
-        assert response.status_code == 200
-        assert 'Tips' in str(response.content)
-
-    def test_tips_views_templates(self):
-        try:
-            get_template(TIPS_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {TIPS_PAGE_ROUTE} does not exist'
-        try:
-            get_template(ADD_TIP_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {ADD_TIP_PAGE_ROUTE} does not exist'
-        try:
-            get_template(EDIT_TIP_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {EDIT_TIP_PAGE_ROUTE} does not exist'
+    def test_tips_views_tips_template(self):
+        get_template(TIPS_PAGE_ROUTE)
```

Comment:
Take a look at my_house_tests file about testing the templates 
At the very least, an assert must be added

Reply:
This is the way it supposed to work
ask @ronyyosef  for more information we allready disccused iit here

Reply:
![image](https://user-images.githubusercontent.com/93659937/170834599-f51c2d0d-cb16-40be-9c7a-e2e888aaae6c.png)